application/requirements.txt

```diff
@@ -12,7 +12,7 @@ httpx-oauth
 kubernetes
```

Comment:
Were is requirement for `httpx`?

Reply:
Line 10

Reply:
Sorry. I forgot that we already used it. And was expecting you to add it in requirements.